services/pkg/sequencer/sequencer.go

```diff
@@ -61,21 +64,26 @@ type MemorySequencer struct {
 	success       map[string][]*model.ActionBatch
 	failure       map[string][]*model.ActionBatch
 	httpClient    *alchemy.Client
+	simHttpClient *alchemy.Client
+	simWsClient   *alchemy.Client
+	simBlock      uint64
 	notifications chan interface{}
+	idxr          indexer.Indexer
 	log           zerolog.Logger
 	sync.RWMutex
 }
 
-func NewMemorySequencer(ctx context.Context, key *ecdsa.PrivateKey, notifications chan interface{}) (*MemorySequencer, error) {
+func NewMemorySequencer(ctx context.Context, key *ecdsa.PrivateKey, notifications chan interface{}, httpProviderURL string, simProviderURL string, idxr indexer.Indexer) (*MemorySequencer, error) {
```

Comment:
I'm trying to find where the simulation chain is actually connected to . I thought this new simProviderURL param would be the path to follow but its unused?

Reply:
good spot ... looks like I was midway through trying to remove the use of the global `config` thing ... willfix

Reply:
fixed ... correctly uses the param now

Reply:
Thanks I've followed the code now through to ResetSim to where the sim urls are used.